test/unit/tech/flash.test.js

```diff
@@ -135,6 +135,16 @@ test('should have the source handler interface', function() {
   ok(Flash.registerSourceHandler, 'has the registerSourceHandler function');
 });
 
+test('canPlayType should select the correct types to play', function () {
+  let canPlayType = Flash.nativeSourceHandler.canPlayType;
+
+  equal('maybe', canPlayType('video/flv'), 'should be able to play FLV files');
```

Comment:
equal's arguments are the reverse of this. The actual is the first argument.
`equal(canPlayType('video/flv'), 'maybe')`